web/ui/react-app/src/pages/LabelsTable.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Button, Badge, Table } from 'reactstrap';
+
+interface LabelProps {
+  value: Record<string, Record<string, string>>[];
+}
+
+export const LabelsTable: FC<RouteComponentProps & LabelProps> = ({ value }) => {
+  const [showMore, doToggle] = useState(false);
+
+  const toggleMore = () => {
+    doToggle(!showMore);
+  };
+
+  return (
+    <>
+      <Button size="sm" color="primary" onClick={toggleMore}>
+        {showMore ? 'More' : 'Less'}
+      </Button>
+      {showMore ? (
+        <Table striped={true} className="table-head">
+          <thead>
+            <tr>
+              <th>Discovered Labels</th>
+              <th>Target Labels</th>
+            </tr>
+          </thead>
+          <tbody>
+            {value.map((_, i) => {
+              return (
+                <tr key={i}>
+                  <td>
+                    {Object.entries(value[i].discoveredLabels).map(([key, value], i) => {
```

Comment:
Suggestion:
```
const printLabels = (labels: Record<string, string>) => {
   return Object.entries(labels).map(([key, value]) => {
        return (
             <div key={key}>
                <Badge color="primary" className='mr-1'>
                     {`${key}="${value}"`}
                 </Badge>
              </div>
          );
    })
}
```
and then you can use it like
```
   <td>
      {printLabels(value[i].discoveredLabels)}
  </td>
   <td>
       {printLabels(value[i].labels)}
   </td>
```

Reply:
done. thanks.